### DISCUSSION TITLE
Deleting cached car_data

### DISCUSSION THREAD

**Discussion by Casper-Guo** [theOehrly/Fast-F1#397] [Q&A] [answered]
I am trying to keep the cache size on my device manageable. Since I do not work with telemetry data, I am considering deleting all the car_data pickles.

Are there any side effects to this or some better ways to reduce cache size?

**Comment by theOehrly** [comment, marked as answer]
> I am trying to keep the cache size on my device manageable. Since I do not work with telemetry data, I am considering deleting all the car_data pickles.
> 
> Are there any side effects to this or some better ways to reduce cache size?

No, you are free to delete any files from the cache directory. In case FastF1 would access the data, it will simply be downloaded again.

Consider using ``session.load(telemetry=False)`` to skip downloading this data in the first place.

**Reply by Casper-Guo** [reply]
Good to know!
